packages/expo-modules-core/android-linker/build.gradle

```diff
@@ -81,6 +81,7 @@ android {
 }
 
 dependencies { dependencyHandler ->
+  implementation project(':expo-modules-core')
```

Comment:
This is a part of #14256, but anyway I'd wait for confirmation from @lukmccall — if I recall correctly this change may result in other issues.